src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindInFilesDialog.java

```diff
@@ -181,6 +187,26 @@ public FindInFilesDialog(OperationWithInput<State> operation)
       setExampleIdAndAriaProperties(spanPatternExample_, txtFilePattern_);
       setExampleIdAndAriaProperties(spanExcludePatternExample_, txtExcludeFilePattern_);
 
+      Roles.getCheckboxRole().setAriaLabelProperty(checkboxCaseSensitive_.getElement(), 
+         checkboxCaseSensitive_.getText());
+      Roles.getCheckboxRole().setAriaLabelProperty(checkboxWholeWord_.getElement(),
+         checkboxWholeWord_.getText());
+      Roles.getCheckboxRole().setAriaLabelProperty(checkboxRegex_.getElement(),
+         checkboxRegex_.getText());
```

Comment:
These three `setAriaLabelProperty` calls are unneeded and should be removed. The CheckBox control already produces proper HTML, in terms of accessibility. ARIA should always be a last resort if semantically correct HTML isn't possible.

More details, the underlying HTML produced by this control is essentially:

```html
<input type="checkbox" id="some_unique_id"/>
<label for="some_unique_id">Visible Label</label>
```

Another way to tell is that if you click on the label text, the checkbox itself will toggle. This is entirely due to the association established by `for` in the label element, and is the preferred native HTML way to do this.

Reply:
Thanks for the explanation about ARIA, that wasn't clear to me. I've updated the code to set ElementId for the elements.